src/eureka/S3_data_reduction/miri.py

```diff
@@ -284,7 +285,16 @@ def fit_bg(dataim, datamask, n, meta, isplots=0):
     n : int
         The current integration number.
     """
-    return nircam.fit_bg(dataim, datamask, n, meta, isplots=isplots)
+    if hasattr(meta, 'isrotate'):
+        isrotate = meta.isrotate
+    else:
+        isrotate = 2
```

Comment:
The data is already rotated by this point in Stage 3, so setting isrotate to 2 by default isn't the correct behaviour, right?

Reply:
`miri.fit_bg()` is now called in Stages 1 and 3.  When called in Stage 3, the default behavior is to use `isrotate = 2` for the reasons mentioned in the conversation above.  When called in Stage 1, the data do not need to be rotated, so the use should be using `meta.isrotate = False`.